lib/src/widgets/form_control_inherited_notifier.dart

```diff
@@ -13,13 +13,13 @@ import 'package:reactive_forms/src/widgets/inherited_streamer.dart';
 ///
 /// The [Listenable] is provided by the [notifierDelegate].
 ///
-class FormControlInheritedStreamer extends InheritedStreamer<dynamic> {
+class FormControlInheritedStreamer extends InheritedStreamer<Object?> {
   final AbstractControl<Object> control;
 
   FormControlInheritedStreamer({
     Key? key,
     required this.control,
-    required Stream<dynamic> stream,
+    required Stream<Object?> stream,
```

Comment:
The same here. I mean, I have rolled back Object to dynamic in many places because of cast exceptions. I don't remember right now all the cases and I would need to search in git history and start remembering. My concern is that changing again to Object? will bring back all those nightmares casts exception.

Reply:
I would argue for some kind of beta release with these changes and correctly fix errors if they present themselves. Keeping dynamic is counter-productive in a null-safe library.